src/main/kotlin/racingcar/communication/ControlTower.kt

```diff
@@ -1,12 +1,25 @@
 package racingcar.communication
 
-import racingcar.domain.racing.FinalLab
 import racingcar.communication.input.Input
 import racingcar.communication.output.Output
+import racingcar.domain.cars.RacingCar
+import racingcar.domain.racing.FinalLab
 
 object ControlTower {
     fun decideRacingLabs(input: Input, output: Output): FinalLab {
         output.finalLab()
         return FinalLab(value = input.finalLab())
     }
+
+    fun judgeTheWinningRecord(racingCarsList: List<RacingCar>): Int =
+        racingCarsList.maxOf { it.racingDistance() }
+
+    fun judgeTheWinnersNames(
+        racingCarsList: List<RacingCar>,
+        winningRecord: Int
+    ) = racingCarsList.filter {
+        it.racingDistance() == winningRecord
+    }.map {
+        it.name
+    }
```

Comment:
```suggestion
    ) = racingCarsList
        .filter {
            it.racingDistance() == winningRecord
        }.map {
            it.name
        }
```
함수 체이닝 컨벤션을 적용해주시면 좋을 것 같아요

Reply:
넵